install-dev/controllers/console/process.php

```diff
@@ -111,6 +113,12 @@ public function initializeContext()
         require_once _PS_ROOT_DIR_ . '/config/smarty.config.inc.php';
 
         Context::getContext()->smarty = $smarty;
+
+        $container = SymfonyContainer::getInstance();
+        if ($container && $container->has(ContextBuilderInitializer::class)) {
```

Comment:
Are there scenarios where the service would not be available?

Reply:
I'd say if the container is not from an AdminKernel maybe 🤔
You think I should remove this check and let the code fail when the service is not found, so that we can explicitly identify these error cases?

Reply:
I think yes, this condition could pose a problem if an error is present much later, because the contexts would not have been setup correctly